src/components/PostCard/PostHeader/PostHeader.js

```diff
@@ -9,24 +9,25 @@ import Highlight from 'components/Highlight'
 import FlagContent from 'components/FlagContent'
 import Icon from 'components/Icon'
 import { personUrl, topicUrl } from 'util/navigation'
+import { formatDatePair } from 'util/index'
 import { humanDate } from 'hylo-utils/text'
 import './PostHeader.scss'
 import { filter, isFunction, isEmpty } from 'lodash'
 import cx from 'classnames'
 
-const formatStartDate = (startTime) => {
+const formatStartDate = (startTime, from) => {
```

Comment:
so this function is not really doing anything anymore, except checking whether the start time is in the future. you should either remove the function, or change it to be a generic one that checks if a date is in the future